Gruntfile.js

```diff
@@ -1637,6 +1652,32 @@ module.exports = function(grunt) {
 		} );
 	} );
 
+	grunt.registerTask( 'packages:update', 'Update WordPress packages', function() {
+		const distTag = grunt.option('dist-tag') || 'latest';
+		grunt.log.writeln( `Updating WordPress packages (--dist-tag=${distTag})` );
+		spawn( 'npx', [ 'wp-scripts', 'packages-update', '--', `--dist-tag=${distTag}` ], {
+			cwd: __dirname,
+			stdio: 'inherit',
+		} );
+	} );
+
+	grunt.registerTask( 'browserslist:update', 'Update WordPress packages', function() {
```

Comment:
@desrosj, how do you feel about running this command every time we refresh WordPress packages from Gutenberg?

I know you open tickets to document the changes for supported browsers.

Reply:
@gziolo this just updates the database of supported features, it doesn't affect the browsers supported by WordPress in any way.

Reply:
I know it’s expected to run this command regularly (https://github.com/browserslist/browserslist/issues/492). It's perfectly fine for `trunk`, but it might not be expected for older major WordPress releases when you want only update packages to fix a security vulnerabilities. 

See also https://core.trac.wordpress.org/ticket/52331 as a reference.

@desrosj opened https://core.trac.wordpress.org/ticket/55559to document supported browser in 6.0 cycle with a note that the list shouldn’t change after RC1.

Reply:
Hm that's a fair point. Let's make this opt-out then, and I'll add a CLI flag to enable it on demand.